docs/integration_quality_scale_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Integration Quality Scale"
+sidebar_label: "Introduction"
+---
+
+The Integration Quality Scale we want to make it clear how well each integration works. Each level of the quality scale consists of a list of requirements. If an integration matches all requirements, it's considered that level.
```

Comment:
"With the Integration Quality Scale we want to make it clear..." 

Or rewrite it further.

Reply:
, it's considered to have reached that level.